### PR TITLE
Include addon status models

### PR DESCRIPTION
**Pull request by ashishmax31**

### CODE REVIEW

File: model/addons_mgmt/v1/addon_clusters_resource.model [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages a list of clusters.
+resource Clusters {
+	// Reference to the specific cluster which an addon can be installed on
+	locator Cluster {
+	    target Cluster
+        variable ID
```
Comment (tzvatot) [inline model/addons_mgmt/v1/addon_clusters_resource.model:22]: Looks like indentation issue

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: model/addons_mgmt/v1/addon_status_type.model [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+// Representation of an addon status.
+class AddonStatus {
+    // Identifier for co-relating current AddonCR revision and reported status.
```
Comment (ciaranRoche) [inline model/addons_mgmt/v1/addon_status_type.model:20]: We are missing the addon_id from here 

https://gitlab.cee.redhat.com/ocm/ocm-addons-service/-/blob/main/openapi/ocm-addons-service.yaml#L769

Comment (ciaranRoche) [reply model/addons_mgmt/v1/addon_status_type.model:20]: looks like that is all that is missing, i am able to create, get, list and delete using the SDK when I added the addon_id

File: model/addons_mgmt/v1/addon_status_type.model [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    CorrelationID String
+
+    // List of reported addon status conditions 
+    AddOnStatusConditions []AddonStatusCondition
```
Comment (ciaranRoche) [inline model/addons_mgmt/v1/addon_status_type.model:24]: Should just be `StatusConditions`

**Comment by ciaranRoche** [comment]
LGTM

@tzvatot PTAL if you are happy with your requested changes, can you merge this please :pray:

**Comment by tzvatot** [comment]
LGTM

**Comment by ciaranRoche** [comment]
JIRA - https://issues.redhat.com/browse/SDA-7381

**Comment by ciaranRoche** [comment]
JIRA - https://issues.redhat.com/browse/SDA-7380

**Pull request merged**